execution/relation.go

```diff
@@ -168,13 +174,74 @@ func (rel *Like) Apply(ctx context.Context, variables octosql.Variables, left, r
 			leftValue.Show(), rightValue.Show(), leftValue.GetType(), rightValue.GetType())
 	}
 
-	match, err := regexp.MatchString(rightValue.AsString(), leftValue.AsString())
+	patternString, err := likePatternToRegexp(rightValue.AsString())
+	if err != nil {
+		return false, errors.Wrapf(err, "couldn't transform LIKE pattern %v to regexp", patternString)
+	}
+
+	match, err := regexp.MatchString(patternString, leftValue.AsString())
 	if err != nil {
 		return false, errors.Wrapf(err, "couldn't match string in like relation with pattern %v", rightValue)
 	}
 	return match, nil
 }
 
+//we assume that the escape character is '\'
+func likePatternToRegexp(pattern string) (string, error) {
+	length := len(pattern)
+	if pattern[length-1] == LikeEscape {
```

Comment:
Not true.

You can have a pattern like this: "\\"

Reply:
You are absolutely right, I will change that :)